examples/src/main/java/com/flipkart/gjex/examples/helloworld/guice/HelloWorldModule.java

```diff
@@ -56,7 +52,7 @@ protected void configure() {
 		bind(Filter.class).annotatedWith(Names.named("LoggingFilter")).to(LoggingFilter.class);
 		bind(Filter.class).annotatedWith(Names.named("AuthFilter")).to(AuthFilter.class);
 		bind(TracingSampler.class).to(AllWhitelistTracingSampler.class);
-		bind(HealthCheck.class).to(AllIsWellHealthCheck.class);
```

Comment:
@kingster Should we remove AllIsWellHealthCheck.java altogether considering we are providing a health check implementation by default as part of server?

Reply:
We can, but then users shouldn't need to bind this. This should be part of the defaults then.

Reply:
removed AllIsWellHealthCheck

Reply:
This should be by default now. Lets move it

Reply:
Now binding of RotationManagementBasedHealthCheck is done in ApiModule